### ISSUE TITLE
Need to show URL extensions

### ISSUE THREAD

**Issue by scottstgelais** [BlogEngine/BlogEngine.NET#239]
I have a blog running an older version of BlogEngine.net that I want to upgrade to either a new blogging app or the last version of BlogEngine.net (3.3.8.0). I have worked out all of the bugs with my theme on 3.3.8.0, but I have run into a problem. I have over 300 posts covering over a decade and would like to have the URL extensions show. Is this possible?

**Comment by scottstgelais** [comment]
I manually changed <removeextensionsfromurls> from True to False in the settings.xml file, but BE 3.3.8.0 is still not showing URL extensions.

**Comment by geekiest** [comment]
Solved this issue ?

**Comment by scottstgelais** [comment]
No. I could only get the posts to redirect to the pages without extensions.
